tasks/pyenv.yml

```diff
@@ -88,3 +88,11 @@
   shell: . /home/{{ odoo_role_odoo_user }}/pyenv/.pyenvrc && pyenv virtualenv {{ odoo_role_python_version  }} {{ odoo_role_venv_name }}
   args:
     creates: "/home/{{ odoo_role_odoo_user }}/pyenv/versions/{{ odoo_role_venv_name }}/bin/python"
+
+- name: Upgrade pip
+  become: true
+  become_user: "{{ odoo_role_odoo_user }}"
+  pip:
+    name: pip
+    state: latest
```

Comment:
Would be better to fix the version to avoid different versions when `pip` gets upgrades between deploys.

Reply:
For instance we could declare a variable to fix the version and if not defined `latest` by default

Reply:
Also, should we specify the `odoo` virtualenv?